torch/_inductor/ir.py

```diff
@@ -3278,50 +3278,28 @@ def _prepare_convolution_fusion_create(
     padding = tuple(padding_)
     dilation = tuple(dilation_)
     assert isinstance(groups, int)
-
+    with FakeTensorMode():
+        output, *_ = cls.process_kernel(
+            torch.ops.aten.convolution,
+            x,
+            weight,
+            bias,
+            stride,
+            padding,
+            dilation,
+            False,
+            [0, 0],
+            groups,
+        )
+
+    output_size = output.shape
     weight_shape = [
         sympy.Integer(V.graph.sizevars.guard_static_shape(s)) for s in weight.get_size()
     ]
-
-    out_channels, in_channels1, *kernel_size = weight_shape
-    in_channels1 = in_channels1 * groups
-    assert len(x.get_size()) == 2 + len(kernel_size)
-    batch, in_channels2, *input_size = x.get_size()
-    output_size = [batch]
-    V.graph.sizevars.guard_equals(in_channels1, in_channels2)
-
-    output_size.append(out_channels)
-    assert (
-        len(stride)
-        == len(padding)
-        == len(dilation)
-        == len(kernel_size)
-        == len(input_size)
+    _, _, *kernel_size = weight.get_size()
```

Comment:
We don't need to get the `kernel_size` from `weight_shape` now?

Reply:
yes, it can directly get the kernel_size from weight_shape, changed.